### ISSUE TITLE
disk formatting would failed on non-NVME drives

### ISSUE THREAD

**Issue by guangbochen** [rancher-sandbox/cOS-toolkit#945]
While testing the fix for NVME drive (https://github.com/rancher-sandbox/cOS-toolkit/pull/933/files), I noticed that sporadically formatting would fail on non-NVME drives:
```
Formatting drives..
1+0 records in
1+0 records out
1048576 bytes (1.0 MB, 1.0 MiB) copied, 0.00275863 s, 380 MB/s
mkfs.fat 4.1 (2017-01-24)
mkfs.vfat: unable to open /dev/vdap1: No such file or directory
```
Seems like it's the partition syncing problem again. If I add partprobe and sleep 5 right after parted command, no errors at all
Adding udevadm settle right after parted command also works cc @johnliu55tw

**Comment by mudler** [comment]
Hi @guangbochen :wave: thanks for opening the issue! I've already opened #938  which is about this one and a fix for it is included in #937

**Comment by guangbochen** [comment]
awesome, I think the GH issue is a better place to track our dependency issue, close in favor of https://github.com/rancher-sandbox/cOS-toolkit/issues/938
